spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersController do
+  it { should route(:get, user_path(1)).to(action: :show, id: 1) }
+  it { should route(:post, signup_path).to(action: :create) }
+
+  describe 'GET #new' do
+    it { expect(response).to have_http_status(:success) }
+  end
+
+  describe 'GET #create' do
+    it { expect(response).to have_http_status(:success) }
+  end
+
+  # describe 'POST #create' do
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂